src/script/conversation/ConversationRepository.ts

```diff
@@ -1351,6 +1353,14 @@ export class ConversationRepository {
     }
   };
 
+  private readonly markConversationReadOnly = async (conversationEntity: Conversation) => {
+    //console.log('conversationEntity', conversationEntity.is_conversation_readonly());
+    conversationEntity.mls1To1ConversationState(true);
```

Comment:
How about being more clear what it represents, e.g `mls1To1ConversationReadOnlyState`?